packages/preview/resume-ng/1.0.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "resume-ng"
+version = "1.0.0"
+entrypoint = "template.typ"
+authors = ["FengKaiyu <https://github.com/fky2015>"]
+license = "MIT"
+description = "A Typst resume designed for optimal information density and aesthetic appeal."
+repository = "https://github.com/fky2015/resume-ng-typst"
+keywords = ["resume", "cv"]
+categories = ["cv"]
+exclude = [".typstignore"]
+
+[template]
+path = "."
```

Comment:
The directory you point to here will be entirely copied when initializing a project using your template. For this reason, it should be a subdirectory of your package, and not its root.

Reply:
I apologize for the incorrect formatting of the project due to a lack of understanding of the readme, the problem has been fixed. Thank you for your guidance.

Reply:
No worries, thank you for fixing it.